Maestro/subscriptions.json

```diff
@@ -846,6 +846,7 @@
         "https://github.com/dotnet/cli-migrate/blob/release/**/*",
         "https://github.com/dotnet/clicommandlineparser/blob/master/**/*",
         "https://github.com/dotnet/clicommandlineparser/blob/release/**/*",
+        "https://github.com/dotnet/command-line-api/blob/master/**/*",
```

Comment:
Any chance for a release branch in this repo? Could preemptively add a trigger for it.